ob-chatgpt-shell.el

```diff
@@ -65,18 +65,27 @@ This function is called by `org-babel-execute-src-block'"
         (cons 'role "system")
         (cons 'content (map-elt params :system)))))
     (when (map-elt params :context)
-      (ob-chatgpt-shell--context))
+      (ob-chatgpt-shell--context (map-elt params :context)))
     `(((role . "user")
        (content . ,body))))
    (map-elt params :version)
    nil nil
    (map-elt params :temperature)))
 
-(defun ob-chatgpt-shell--context ()
-  "Return the context (what was asked and responded) in all previous blocks."
+(defun ob-chatgpt-shell--context (context-name)
```

Comment:
TODO: Continue to support cases where `context` is just the boolean `t` -- we can have both!